### DISCUSSION TITLE
maftools oncoplot samples

### DISCUSSION THREAD

**Discussion by WilliamPolar** [ShixiangWang/self-study#116] [General]
Dr. Wang, Thank you for your help.
Why is it that the Oncoplot is showing only 68 samples out of 73 samples from the maf?
It must be a simple answer but I cannot troubleshoot.

**Comment by ShixiangWang** [comment]
Hi @WilliamPolar , Please try setting `removeNonMutated = FALSE` in the plotting function, check https://rdrr.io/bioc/maftools/man/oncoplot.html

**Reply by WilliamPolar** [reply]
Is there a way to add continuous value annotation instead of categorical?
For example, If I have a score I generated that is a range of numbers, I want to show the spectrum, I guess I can give definition for each number too but I wonder if that is something this package has or I will just have to give each a definition.

**Reply by ShixiangWang** [reply]
Please see https://rdrr.io/bioc/maftools/f/vignettes/oncoplots.Rmd 

I suggest you raise these questions in maftools GitHub repo https://github.com/PoisonAlien/maftools.

**Comment by WilliamPolar** [comment]
Thank you so much! this was so helpful~
